transcription/google_vision.py

```
@@ -0,0 +1,181 @@
+    if image_path.lower().endswith((".png", ".jpg", ".jpeg")):
+        # check image resolution 
+        # document ai has a restriction on image resolution to 40 megapixels per page
+        #image_utils.resize_image(image_path)
+
+        try:
+            # document ai processing 
+            extracted_text = batch_process_documents(image_path, "image/jpeg")
+            processed_text = generate_metadata(extracted_text, shots)
+
+            # build result 
+            result = eval(processed_text)
```

Ruff (0.14.8)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)